extensions/wikia/ContentReview/controllers/ContentReviewSpecialController.class.php

```diff
@@ -59,6 +59,9 @@ private function prepareReviewData( $reviewsRaw ) {
 					? wfMessage( 'content-review-special-start-review' )->escaped()
 					: wfMessage( 'content-review-special-continue-review' )->escaped();
 
+				if ( !empty( $review['review_user_id'] ) ) {
+					$review['review_user_id'] = User::newFromId( $review['review_user_id'] )->getName();
```

Comment:
Since it's not a `user_id` anymore we should rename the key e.g. to `reviewer_user_name`